webservices/args.py

```diff
@@ -231,6 +231,7 @@ def make_seek_args(field=fields.Int, description=None):
 
 totals = {
     'cycle': fields.List(fields.Int, description=docs.RECORD_CYCLE),
+    'type': fields.List(fields.Str, description=docs.COMMITTEE_TYPE)
```

Comment:
I think we don't need this because committee type is passed in as part of the url here https://github.com/18F/openFEC/pull/1772/files#diff-36f6a3b25691ff374f1941da4493c150L27